examples/build_all.sh

```diff
@@ -7,31 +7,102 @@ function ctrl_c() {
     kill $PID
 }
 
-function build() {
+function build_std_web() {
     for example in */ ; do
         if [[ $example == server* ]]; then
             continue
         fi
-        echo "Building: $example"
-        cd $example
-        cargo update
-        cargo web build --target wasm32-unknown-unknown
-        cd ..
+        if [[ $example == static* ]]; then
```

Comment:
Is this needed?

Reply:
Well we need to skip this folder because it doesn't contain any valid crate, like with `server`, otherwise it would error.
It's the same piece of code as above with the `server` folder.

Reply:
I see, didn't realize you added the `static/` folder, looks fine!